README.md

```diff
@@ -215,6 +215,25 @@ IncidentAssert assertions = BpmnAssert.assertThat(activatedJob)
   .extractingLatestIncident();
 ```
 
+### Custom Mapper
+
+Users can now provide their own `ObjectMapper` to the `Zeebe Client`
```

Comment:
🔧 Instead of saying users can _now_ use it, let's keep it independent from a time. Something along the lines of

"A custom `ObjecctMapper` can be provided" and then explaining how to do that.